_data/sidebars/pbc_all_sidebar.yml

```diff
@@ -39,5 +39,57 @@ entries:
                 url: /docs/pbc/all/payment-service-providers/unzer/domain-model-and-relationships.html
           - title: Payone
             url: /docs/pbc/all/payment-service-providers/payone/payone.html
+      - title: Tax Management
+        url: /docs/pbc/all/tax-management/tax-management.html
+        nested:
+          - title: Install and upgrade
+            nested:
+              - title: Integrate the Tax Glue API
+                url: /docs/pbc/all/tax-management/install-and-upgrade/integrate-the-tax-glue-api.html
+              - title: Integrate the Product Tax Sets Glue API
+                url: /docs/pbc/all/tax-management/install-and-upgrade/integrate-the-product-tax-sets-glue-api.html
+              - title: Upgrade the ProductTaxSetsRestApi module
+                url: /docs/pbc/all/tax-management/install-and-upgrade/upgrade-the-producttaxsetsrestapi-module.html
+              - title: Upgrade the Tax module
+                url: /docs/pbc/all/tax-management/install-and-upgrade/upgrade-the-tax-module.html
+          - title: Manage in the Back Office
+            nested:
+              - title: Create tax rates
+                url: /docs/pbc/all/tax-management/manage-in-the-back-office/create-tax-rates.html
+              - title: Edit tax rates
+                url: /docs/pbc/all/tax-management/manage-in-the-back-office/edit-tax-rates.html
+              - title: Create tax sets
+                url: /docs/pbc/all/tax-management/manage-in-the-back-office/create-tax-sets.html
+              - title: Edit tax sets
+                url: /docs/pbc/all/tax-management/manage-in-the-back-office/edit-tax-sets.html
+          - title: Manage via Glue API
+            nested:
+              - title: Retrieve tax sets
+                url: /docs/pbc/all/tax-management/manage-via-glue-api/retrieve-tax-sets.html
+              - title: Retrieve tax sets of abstract products
+                url: /docs/pbc/all/tax-management/manage-via-glue-api/retrieve-tax-sets-of-abstract-products.html                
+          - title: Import and export data
+            nested:
+              - title: Import tax sets
+                url: /docs/pbc/all/tax-management/data-import-and-export/import-tax-sets.html
+              - title: Import tax sets for abstract products
+                url: /docs/pbc/all/tax-management/data-import-and-export/import-tax-sets-for-abstract-products.html
+          - title: Extend and customize
+            nested:
+              - title: "Tax module: reference information"
+                url: /docs/pbc/all/tax-management/extend-and-customize/tax-module-reference-information.html
+          - title: Domain model and relationships
+            nested:
+              - title: Tax entity diagrams
+                url: /docs/pbc/all/tax-management/domain-model-and-relationships/tax-entity-digrams.html
```

Comment:
The domain model should not be inside a folder, it is just a page. Check how it was done for Unzer